data-prepper-api/src/test/java/com/amazon/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -310,13 +312,21 @@ public void testToString_withSimpleObject() {
         event.put("list", Arrays.asList(1, 4, 5));
         final String result = event.toJsonString();
 
-        assertThat(result, is(equalTo(String.format("{\"foo\":\"bar\",\"testObject\":{\"field1\":\"%s\"},\"list\":[1,4,5]}", value))));
+        String expectedResult = "{\"event_timestamp\":\"" + JacksonEvent.getZonedDateTime() +
+                "\",\"foo\":\"bar\",\"testObject\":{\"field1\":\"%s\"},\"list\":[1,4,5]}";
+        assertThat(result, equalTo(String.format(expectedResult, value)));
     }
 
     @Test
     public void testGetAsMap_with_EmptyData() {
         final Map<String, Object> eventAsMap = event.toMap();
-        assertThat(eventAsMap, equalTo(Collections.emptyMap()));
+
+        final Map<String, Object> mapObject = new HashMap<>();
+        mapObject.put("event_timestamp", JacksonEvent.getZonedDateTime());
```

Comment:
You should create your ZonedDateTime in this test. It can be in either the setUp or in this method. You don't need that static method on `JacksonEvent`.